docs/references/nextjs/clerk-middleware.mdx

```diff
@@ -336,38 +336,45 @@ The `clerkMiddleware()` function accepts an optional object. The following optio
 
   ---
 
+  - `organizationSyncOptions?`
+  - <code>[OrganizationSyncOptions](#organization-sync-options) | undefined</code>
+
+  Used to activate a particular [organization](docs/organizations/overview) or the [personal account](/glossary#personal-account) based on path parameters in the URL.
+
+  ---
+
   - `proxyUrl?`
   - `string`
 
   Specify the URL of the proxy, if using a proxy.
 
   ---
 
-  - `signInUrl?`
+  - `publishableKey`
   - `string`
 
-  An alternative sign in URL.
+  The Clerk publishable key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.
 
   ---
 
-  - `signUpUrl?`
+  - `secretKey?`
   - `string`
 
-  An alternative sign up URL.
+  The Clerk secret key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page. The `CLERK_ENCRYPTION_KEY` environment variable must be set when providing `secretKey` as an option, refer to [Dynamic keys](#dynamic-keys).
 
   ---
 
-  - `publishableKey`
+  - `signInUrl?`
   - `string`
 
-  The Clerk publishable key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.
+  An alternative sign in URL.
 
   ---
 
-  - `secretKey?`
+  - `signUpUrl?`
   - `string`
 
-  The Clerk secret key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page. The `CLERK_ENCRYPTION_KEY` environment variable must be set when providing `secretKey` as an option, refer to [Dynamic keys](#dynamic-keys).
+  An alternative sign up URL.
```

Comment:
This is all just a re-arrangement - I noticed that the list was _almost_ alphabetized, and now it's alphabetized.